generators/chipyard/src/main/scala/harness/HarnessBinders.scala

```diff
@@ -252,6 +252,13 @@ class WithSimTSIOverSerialTL extends HarnessBinder({
   }
 })
 
+//TODO: Set with chipId argument to harness binder, hardcoding is temporary hack
```

Comment:
Remove comment.

Also, can we check that port.io.getWidth is enough for whatever chipId is? Want to avoid cases where the chipid width is too narrow for the desired nChips.